modules/gitops-configuring-gitops-managed-resources-to-use-mounted-secrets.adoc

```diff
@@ -0,0 +1,117 @@
+// Module is included in the following assemblies:
+//
+// * securing_openshift_gitops/managing-secrets-securely-using-sscsid-with-gitops.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-configuring-gitops-managed-resources-to-use-mounted-secrets_{context}"]
+= Configuring {gitops-shortname} managed resources to use mounted secrets
+
+You must configure the {gitops-shortname} managed resources by adding volume mounts configuration to a deployment and configure the container pod to use the mounted secret. For example, consider that you want to add volume mounts configuration to the deployment of `app-taxi` application and the `100-deployment.yaml` file is in the `/environments/dev/apps/app-taxi/services/taxi/base/config/` directory.
+
+.Prerequisites
+
+* You have the AWS Secrets Manager resources stored in your {gitops-shortname} repository.
+* You have the Secrets Store Container Storage Interface (SSCSI) driver configured to mount secrets from AWS Secrets Manager.
+
+.Procedure
+
+. Add volume mounts configuration in the target resource:
+
+.. Create a `100-deployment.yaml` file to use the secret provider class resources:
++
+.Example `100-deployment.yaml` file
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  creationTimestamp: null
+  labels:
+    app.kubernetes.io/name: taxi
+    app.kubernetes.io/part-of: app-taxi
+    name: taxi                                # <1>
+    namespace: dev                            # <2>
```

Comment:
I had some indent issue from my documentation, `name` and `namespace` are under `metadata`, instead of `metadata.labels`. We can remove #<1> as it's quite obviously what name it is.

```
...
metadata:
  creationTimestamp: null
  labels:
    app.kubernetes.io/name: taxi
    app.kubernetes.io/part-of: app-taxi
  name: taxi
  namespace: dev                            # <1>
```

Reply:
Addressed